### PR TITLE
Fix: Remove unused property

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] removes an unused property

### CODE REVIEW

File: src/Downloader.php [hunk 1]
```diff
@@ -51,7 +50,6 @@ public function __construct(Composer $composer, IoInterface $io, ParallelDownloa
         $this->endpoint = rtrim($endpoint, '/');
         $this->io = $io;
         $config = $composer->getConfig();
-        $this->eventDispatcher = $composer->getEventDispatcher();
```
Comment (localheinz) [inline src/Downloader.php]: Property is assigned a value, but never used.

**Comment by fabpot** [comment]
Next time, I would prefer one PR for "simple" changes, that's better for :)

**Comment by fabpot** [comment]
Good catch, thanks @localheinz.

**Comment by localheinz** [comment]
Thank you, @derrabus, @fabpot, and @xabbuh!

**Pull request merged**